public/js/charts/worstPMGraph.js

```diff
@@ -0,0 +1,145 @@
+// NOTE: requires('sheets');
+// NOTE: requires('aq');
+
+window.aq.charts.worstNodes = {};
+
+const ctx = document.getElementById('myChart').getContext('2d');
+
+function readingAverage(dataValues) {
+  // get total of AQ recordings
+  const recordings = dataValues.length;
+
+  let average = 0;
+  let sum = 0;
+
+  dataValues.forEach((value) => {
+    // loop through the values and sum the total of the P1,2 and 0 values
+    if (
+      value.value_type === 'P2' ||
+      value.value_type === 'P1' ||
+      value.value_type === 'P0'
+    ) {
+      // sum them up
+      sum += Number(value.value);
+    }
+  });
```

Comment:
You can't do this; they're different measurements. You have to sum each one individually.

Reply:
ahh, okay. So with that in mind then a node should return 3 averages? For P1, P2 and P0? @kilemensi

Reply:
Yes!

Reply:
Great, so if a node is returning 3 averages, then what's to be displayed on the chart?  The worst of P1? The worst of P2?  The worst of the three? @kilemensi

Reply:
For simplicity, we can pick one and show that @gertie-sheshe ... e.g. P2 only.

But if possible, you can show each individual measurement as we did in the [Data4SDGs](https://data4sdgs.sensors.africa/dashboard)